packages/cojson/package.json

```diff
@@ -7,13 +7,13 @@
     "exports": {
         ".": {
             "react-native": "./dist/native/index.native.js",
-            "default": "./dist/web/index.web.js",
-            "types": "./src/index.web.ts"
+            "types": "./src/index.web.ts",
+            "default": "./dist/web/index.web.js"
```

Comment:
@aeplay @pax-k @bensleveritt @gdorsi 
I was dealing with a build error `Module not found: Error: Default condition should be last one` and going a bit insane, and turns out the fix is to put `default` last here lol I hope this is ok

Reply:
That should be ok!

Didn't know that the order matters in the exports field! 🤯